### ISSUE TITLE
Can't remove a parameter after refactoring an Actionword

### ISSUE THREAD

**Issue by GoudeQuassi** [hiptest/hiptest-issue-tracker#137]
### Steps to reproduce
1. Create an action word with 2 or more parameters
2. Use the action word in 2 scenarios and assign parameters to Datatable values
3. Remove a parameter from the Action Word and confirm that you wan't to change both scenarios

See https://app.hiptest.com/projects/62692/test-plan/folders/557886/scenarios/1929250
### Expected behaviour
The parameter disappear from the Action word and from both scenarios.

### Actual behaviour
Parameters disappear from the Action word definition, but the parameter remains in both scenarios as extra parameter/button value. Value can be edited, a remove action is available buy can't actually remove the extra parameter.

### My configuration
Firefox60

**Operating system:**
Windows 7 Professional
![bughiptest](https://user-images.githubusercontent.com/6573082/41036889-d861a63a-6991-11e8-931b-c8eea30ce6cf.png)

**Comment by GoudeQuassi** [comment]
It appears that the parameter is still in the Action Word Parameter Table, and can't be reset or removed!

**Comment by vincent-psarga** [comment]
Hi and thank you for the feedback.

I tried to reproduce your issue but I'm not sure what you meant in step 3: "Remove a parameter from the Action Word and confirm that you wan't to change both scenarios"

How did you remove the parameter ? Using the editor inside the scenario and using the option "Rename here" ?
That's what I tried to do. Indeed the parameter was still showing up but I was able to remove the parameter from the table parameter (using the "..." menu then "Remove").
